elastic/apm-server
# apmpackage: Update listen socket to `0.0.0.0:8200`

## Motivation/summary

The APM integration currently will listen on localhost:8200 which won't
receive any of the traffic sent to that address since it's attached to
`localhost` instead of `0.0.0.0`.

## Checklist

~- [ ] Documentation has been updated~

## How to test these changes

1. `docker-compose up -d`
2. Navigate to `Kibana > Observability > APM > Settings > Schema > Switch to Elastic Agent`
3. Verify that the APM Integration host is set to 0.0.0.0:8200
